pagerduty/resource_pagerduty_automation_actions_action.go

```diff
@@ -298,6 +308,10 @@ func resourcePagerDutyAutomationActionsActionRead(d *schema.ResourceData, meta i
 			if automationActionsAction.ActionClassification != nil {
 				d.Set("action_classification", &automationActionsAction.ActionClassification)
 			}
+
+			if automationActionsAction.OnlyInvocableOnUnresolvedIncidents != nil {
+				d.Set("only_invocable_on_unresolved_incidents", &automationActionsAction.OnlyInvocableOnUnresolvedIncidents)
```

Comment:
As `automationActionsAction.OnlyInvocableOnUnresolvedIncidents` is a pointer to bool the value should be its dereference (i.e. use `*` instead of `&`).